### ISSUE TITLE
Enzyme `wrappingComponent` doesn't work

### ISSUE THREAD

**Issue by princed** [albertogasparin/react-magnetic-di#7]
This recommended form for enzyme doesn't actually work for some reason:

```ts
  const container = mount(<MyComponent />, {
    wrappingComponent: DiProvider,
    wrappingComponentProps: { use: [ModalOpen, useQuery] },
  });
```

No errors, the context is just not provided. 

Works okay if you do this though:

```ts
const container = mount(
  <DiProvider use={[ModalOpen, useQuery]}>
    <MyComponent />
  </DiProvider>
);
```

**Comment by albertogasparin** [comment]
Thank you for trying it out. I has that API in mind but didn't wrote actual code to test it out, so probably there was a bug somewhere.
I've now published v2.0.1 and added [examples for this pattern in unit tests](https://github.com/albertogasparin/react-magnetic-di/tree/master/examples/flow/components/__tests__/input-enzyme.test.js) and I see it working fine.
Can you try it out again and let me know?

PS: different case for `shallow`, where Enzyme does not support `useContext` properly 😞

**Comment by princed** [comment]
Yes, it seems to be working now! Thank you Alberto!
